src/structures/Message.js

```diff
@@ -541,18 +541,27 @@ class Message extends Base {
 
   /**
    * Edits the content of the message.
-   * @param {?(string|APIMessage)} [content] The new content for the message
-   * @param {MessageEditOptions|MessageEmbed|MessageAttachment|MessageAttachment[]} [options] The options to provide
+   * @param {string|APIMessage|MessageEditOptions|MessageEmbed
+   * |MessageAttachment|MessageAttachment[]} options The options to provide
```

Comment:
Not sure if `MessageEmbed | MessageAttachment | MessageAttachment[]` should be here, wouldn't this imply that `message.edit(embed)` works?

Reply:
Currently it does work, that's what I was trying to point out in https://github.com/discordjs/discord.js/pull/5758#issuecomment-855452913 So removing these from typings and docs can work, but leaves undocumented behavior (which people shouldn't use, but will)

Reply:
Yes, I was implying that this is undesired behaviour and should be fixed/removed (not only the docstring).

Reply:
> Yes, I was implying that this is undesired behaviour and should be fixed/removed (not only the docstring).

Have done so within the latest commits